aiken-contracts/fractional/README.md

```diff
@@ -55,21 +55,43 @@ pub type Datum {
 }
 ```
 
-The bundle token information needs to match what is on the UTxO and the cost token information can be whatever token the seller wishes to be paid in. The maximum bundle size is a hardcoded maximum of 10 bundles. This is an arbitrary pick and can be changed to whatever is needed in production.
+The bundle token information needs to match what is on the UTxO and the cost token information can be whatever token the seller wishes to be paid in for a completed sale.
 
-A bundle sale with a bundle size of 100 and a cost of 10 ADA.
+### Example Sale
+A bundle sale with a bundle size of 100 tokens and a cost of 10 ADA. On the UTxO for the sale there will be
 
 ```bash
 1000 policy.token_name
 ```
 
-A buyer wants 6 bundles. They will receive 600 tokens at a cost of 60 ADA.
+A buyer wants 6 bundles. They will receive 600 tokens at a cost of 60 ADA. The remaining 400 tokens will be sent back to the contract for another sale and the seller will receive 60 ADA.
```

Comment:
How do we handle utxo contention when two buyers are trying to spend the same utxo?

Reply:
going to fork off of this branch and do the queue and bank system.